hironx_ros_bridge/test/test_hironx_controller.py

```diff
@@ -1,43 +1,113 @@
         if not self.robot.ic or self.robot.ic.ref.get_component_profile().version < '315.3.0':
             self.assertTrue(True)
             return True
         # although this is not stable rtc, we'll test them
         self.robot.goInitial(tm=1)
-        ret = self.robot.startImpedance('rarm') # this returns ret, this is bug
-        #self.assertTrue(ret)
-        ret = self.robot.seq_svc.setWrenches([0,0,0,0,0,0,
-                                              10,0,0,0,0,0,], 2.0);
-        #self.assertTrue(ret)
+        ret = self.robot.startImpedance('rarm', **kwargs)  # this returns ret, this is bug
+        # self.assertTrue(ret)
+        ret = self.robot.seq_svc.setWrenches([0, 0, 0, 0, 0, 0,
+                                              10, 0, 0, 0, 0, 0, ], 2.0)
+        # self.assertTrue(ret)
         self.robot.seq_svc.waitInterpolation();
-        ret = self.robot.seq_svc.setWrenches([0,0,0,0,0,0,
-                                              0,0,0,0,0,0,], 2.0);
-        #self.assertTrue(ret)
+        ret = self.robot.seq_svc.setWrenches([0, 0, 0, 0, 0, 0,
+                                              0, 0, 0, 0, 0, 0, ], 2.0)
+        # self.assertTrue(ret)
         self.robot.seq_svc.waitInterpolation();
-        ret = self.robot.seq_svc.setWrenches([0,0,0,0,0,0,
-                                              0,10,0,0,0,0,], 2.0);
-        #self.assertTrue(ret)
+        ret = self.robot.seq_svc.setWrenches([0, 0, 0, 0, 0, 0,
+                                              0, 10, 0, 0, 0, 0, ], 2.0)
+        # self.assertTrue(ret)
         self.robot.seq_svc.waitInterpolation();
-        ret = self.robot.seq_svc.setWrenches([0,0,0,0,0,0,
-                                              0,0,0,0,0,0,], 2.0);
-        #self.assertTrue(ret)
+        ret = self.robot.seq_svc.setWrenches([0, 0, 0, 0, 0, 0,
+                                              0, 0, 0, 0, 0, 0, ], 2.0)
+        # self.assertTrue(ret)
         self.robot.seq_svc.waitInterpolation();
-        ret = self.robot.seq_svc.setWrenches([0,0,0,0,0,0,
-                                              0,0,10,0,0,0,], 2.0);
-        #self.assertTrue(ret)
-        self.robot.seq_svc.waitInterpolation();
-        #self.assertTrue(ret)
+        ret = self.robot.seq_svc.setWrenches([0, 0, 0, 0, 0, 0,
+                                              0, 0, 10, 0, 0, 0, ], 2.0)
+        # self.assertTrue(ret)
+        self.robot.seq_svc.waitInterpolation()
+        # self.assertTrue(ret)
         ret = self.robot.stopImpedance('rarm')
-        #self.assertTrue(ret)
-        self.assertTrue(True) # this is dummy, current simulate hiro does not have force sensor so it retunrs None
+        # self.assertTrue(ret)
+        self.assertTrue(True)  # this is dummy, current simulate hiro does not have force sensor so it retunrs None
+
+    def test_impedance_controller_refforce_315_1(self):
```

Comment:
tl;dr I'm leaning toward removing this `test_impedance_controller_refforce_315_1` testcase.

In the test on 315.1.10 ([example](https://s3.amazonaws.com/archive.travis-ci.org/jobs/228585681/log.txt?X-Amz-Expires=30&X-Amz-Date=20170504T033750Z&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJRYRXRSVGNKPKO5A/20170504/us-east-1/s3/aws4_request&X-Amz-SignedHeaders=host&X-Amz-Signature=4aafa58c4e4a3054ff3cd3cd0750c7db678a29445557d174db7718cb7c3c7b45)),

```
[ROSTEST]-----------------------------------------------------------------------

[hironx_ros_bridge.rosunit-test_hironx_controller/test_hands_controller][passed]
[hironx_ros_bridge.rosunit-test_hironx_controller/test_impedance_controller][passed]
[hironx_ros_bridge.rosunit-test_hironx_controller/test_impedance_controller_refforce_315_1][ERROR]
CORBA.BAD_PARAM(omniORB.BAD_PARAM_IndexOutOfRange, CORBA.COMPLETED_NO)
  File "/usr/lib/python2.7/unittest/case.py", line 331, in run
    testMethod()
  File "/home/travis/catkin_ws/install_isolated/share/hironx_ros_bridge/test/test_hironx_controller.py", line 91, in test_impedance_controller_refforce_315_1
    self.robot.startImpedance(KIN_GROUP)
  File "/home/travis/catkin_ws/install_isolated/lib/python2.7/dist-packages/hironx_ros_bridge/hironx_client.py", line 1095, in startImpedance
    self.startImpedance_315_1(arm, **kwargs)
  File "/home/travis/catkin_ws/install_isolated/lib/python2.7/dist-packages/hironx_ros_bridge/hironx_client.py", line 934, in startImpedance_315_1
    manipulability_limit = manipulability_limit))
  File "/home/travis/catkin_ws/install_isolated/lib/python2.7/dist-packages/hironx_ros_bridge/hrpsys_315_1_9/hrpsys/ImpedanceControllerService_idl.py", line 95, in setImpedanceControllerParam
    return _omnipy.invoke(self, "setImpedanceControllerParam", _0_OpenHRP.ImpedanceControllerService._d_setImpedanceControllerParam, args)
```

Although `self.robot.startImpedance('rarm')` can run on the real robot that runs on hrpsys `315.1` (confirmed on 315.1.9), it's the same command that seems to be failing on Travis.

When tested locally the same testcase didn't appear to be failing (despite many others failed).

```
$ catkin run_tests hironx_ros_bridge --no-deps
:
$ catkin_test_results 
build/hironx_ros_bridge/test_results/hironx_ros_bridge/rostest-test_test-hironx-ros-bridge-send.xml: 1 tests, 1 errors, 0 failures, 0 skipped
build/hironx_ros_bridge/test_results/hironx_ros_bridge/rostest-test_test-hironx-ros-bridge.xml: 2 tests, 1 errors, 0 failures, 0 skipped
build/hironx_ros_bridge/test_results/hironx_ros_bridge/rosunit-joint_states_test.xml: 1 tests, 0 errors, 1 failures, 0 skipped
build/hironx_ros_bridge/test_results/hironx_ros_bridge/rosunit-test_hironx_fullbody.xml: 0 tests, 1 errors, 0 failures, 0 skipped
build/hironx_ros_bridge/test_results/hironx_ros_bridge/rosunit-test_hironx_ik.xml: 4 tests, 0 errors, 2 failures, 0 skipped
build/hironx_ros_bridge/test_results/hironx_ros_bridge/rosunit-test_hironx_ik_noinit.xml: 0 tests, 1 errors, 0 failures, 0 skipped
build/hironx_ros_bridge/test_results/hironx_ros_bridge/rosunit-test_hironx_ros_bridge_controller.xml: 1 tests, 0 errors, 1 failures, 0 skipped
build/hironx_ros_bridge/test_results/hironx_ros_bridge/rosunit-test_hironx_ros_bridge_pose.xml: 0 tests, 1 errors, 0 failures, 0 skipped
build/nextage_calibration/test_results/nextage_calibration/rosunit-nxo_calib.xml: 1 tests, 0 errors, 1 failures, 0 skipped
Summary: 38 tests, 5 errors, 5 failures, 0 skipped
```